src/modules/colorPicker/ColorPickerUI/ViewModels/MainViewModel.cs

```diff
@@ -52,116 +65,180 @@ public class MainViewModel : ViewModelBase, IMainViewModel
-                    colorRepresentationToCopy = RgbColor;
-                    break;
-                default:
-                    break;
-            }
-
-            CopyToClipboard(colorRepresentationToCopy);
+            CopyToClipboard(ColorText);
 
             _appStateHandler.HideColorPicker();
             PowerToysTelemetry.Log.WriteEvent(new ColorPickerShowEvent());
         }
 
-        private static void CopyToClipboard(string colorRepresentationToCopy)
+        /// <summary>
+        /// Copy the given text to the Windows clipboard
+        /// </summary>
+        /// <param name="text">The text to copy to the Windows clipboard</param>
+        private static void CopyToClipboard(string text)
         {
-            if (!string.IsNullOrEmpty(colorRepresentationToCopy))
+            if (string.IsNullOrEmpty(text))
+            {
+                return;
+            }
+
+            // nasty hack - sometimes clipboard can be in use and it will raise and exception
+            for (var i = 0; i < 10; i++)
             {
-                // nasty hack - sometimes clipboard can be in use and it will raise and exception
-                for (int i = 0; i < 10; i++)
+                try
                 {
-                    try
-                    {
-                        Clipboard.SetText(colorRepresentationToCopy);
-                        break;
-                    }
-                    catch (COMException ex)
+                    Clipboard.SetText(text);
+                    break;
+                }
+                catch (COMException ex)
+                {
+                    if ((uint)ex.ErrorCode != ErrorCodeClipboardCantOpen)
                     {
-                        const uint CLIPBRD_E_CANT_OPEN = 0x800401D0;
-                        if ((uint)ex.ErrorCode != CLIPBRD_E_CANT_OPEN)
-                        {
-                            Logger.LogError("Failed to set text into clipboard", ex);
-                        }
+                        Logger.LogError("Failed to set text into clipboard", ex);
                     }
-
-                    System.Threading.Thread.Sleep(10);
                 }
+
+                Thread.Sleep(10);
             }
         }
 
+        /// <summary>
+        /// Tell the color picker that the user have used the mouse wheel
+        /// </summary>
+        /// <param name="sender">The sender of this event</param>
+        /// <param name="e">The new values for the zoom</param>
         private void MouseInfoProvider_OnMouseWheel(object sender, Tuple<Point, bool> e)
+            => _zoomWindowHelper.Zoom(e.Item1, e.Item2);
+
+        /// <summary>
+        /// Return a hexadecimal representation of a RGB color
+        /// </summary>
+        /// <param name="color">The color for the hexadecimal presentation</param>
+        /// <returns>A hexadecimal representation of a RGB color</returns>
+        private static string ColorToHex(System.Drawing.Color color)
```

Comment:
Move all these color conversions into some Helpers class, such as `ColorFormatHelper ` so it is not polluting `MainViewModel`.
Make one public static function in that helper class such as 
`string GetStringRepresentation(Color color, ColorFormatRepresentation representation) ` and call it from here where needed

Reply:
Move all representations to separate helper class and add a UnitTest for it.
@martinchrzan is there a reason why you say the function should be `public` instead of `internal` ???

Reply:
no, internal is fine, I just meant that only one method should be accessible from outside from that helper class, other ones should stay private.